docs/root/configuration/http/http_filters/composite_filter.rst

```diff
@@ -5,8 +5,7 @@ Composite Filter
 
 .. attention::
 
-   The composite filter is in alpha and is currently under active development.
-   Capabilities will be expanded over time and the configuration structures are likely to change.
+   The composite filter is in stable status. Breaking API changes are not allowed.
```

Comment:
I think the entire `attention` section can be removed (if no comment appears, it is stable by default).

Reply:
Sure. yea, I was wondering if I should just get rid of it.